irony-cdb-libclang.el

```diff
@@ -0,0 +1,89 @@
+;;; irony-cdb-libclang.el --- Compilation Database for irony using libclang
+
+;; Copyright (C) 2014  Guillaume Papin
+
+;; Author: Guillaume Papin <guillaume.papin@epitech.eu>
+;; Keywords: c, convenience, tools
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;
+;; Compilation Database support for Irony using libclangs CXCompilationDatabase.
+;; Currently libclang only supports json compilation databases, but reading
+;; using libclang is much faster than using json.el
+
+;;; Code:
+
+(require 'irony-cdb)
+(require 'cl-lib)
```

Comment:
nitpicking:

I add a newline between irony's own `require` "local requires" and "third-party requires", and built-in requires, like I do in C++ with includes.

So I would favor:

```el
(require 'irony-cdb)

(require 'cl-lib)
```